src/v2/Apps/Settings/Routes/DeleteAccount/DeleteAccountRoute.tsx

```diff
@@ -90,6 +116,20 @@ export const DeleteAccountRoute: FC = () => {
                   }}
                 />
 
+                {hasPassword && (
+                  <PasswordInput
+                    error={touched.password && errors.password}
+                    mt={2}
+                    name="password"
+                    onBlur={handleBlur}
```

Comment:
w/o this the UI doesn't display validation error at all. w/ this, there's still no validation error when you first start typing in the field, but once you click out of the field, the error shows.

Reply:
the non-social gif demonstration shows the behavior.

Reply:
Interesting, 👀

Reply:
👍